src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf

```diff
@@ -472,87 +472,87 @@
+                <target>Ce fichier n’est pas une vidéo valide.</target>
             </trans-unit>
             <trans-unit id="123">
                 <source>The size of the video could not be detected.</source>
-                <target state="needs-review-translation">La taille de la vidéo n’a pas pu être détectée.</target>
+                <target>La taille de la vidéo n’a pas pu être détectée.</target>
             </trans-unit>
             <trans-unit id="124">
                 <source>The video width is too big ({{ width }}px). Allowed maximum width is {{ max_width }}px.</source>
-                <target state="needs-review-translation">La largeur de la vidéo est trop grande ({{ width }}px). La largeur maximale autorisée est de {{ max_width }}px.</target>
+                <target>La largeur de la vidéo est trop grande ({{ width }}px). La largeur maximale autorisée est de {{ max_width }}px.</target>
             </trans-unit>
             <trans-unit id="125">
                 <source>The video width is too small ({{ width }}px). Minimum width expected is {{ min_width }}px.</source>
-                <target state="needs-review-translation">La largeur de la vidéo est trop petite ({{ width }}px). La largeur minimale attendue est de {{ min_width }} px.</target>
+                <target>La largeur de la vidéo est trop petite ({{ width }}px). La largeur minimale attendue est de {{ min_width }}px.</target>
             </trans-unit>
             <trans-unit id="126">
                 <source>The video height is too big ({{ height }}px). Allowed maximum height is {{ max_height }}px.</source>
-                <target state="needs-review-translation">La hauteur de la vidéo est trop grande ({{ height }}px). La hauteur maximale autorisée est de {{ max_height }}px.</target>
+                <target>La hauteur de la vidéo est trop grande ({{ height }}px). La hauteur maximale autorisée est de {{ max_height }}px.</target>
             </trans-unit>
             <trans-unit id="127">
                 <source>The video height is too small ({{ height }}px). Minimum height expected is {{ min_height }}px.</source>
-                <target state="needs-review-translation">La hauteur de la vidéo est trop petite ({{ height }}px). La hauteur minimale attendue est de {{ min_height }}px.</target>
+                <target>La hauteur de la vidéo est trop petite ({{ height }}px). La hauteur minimale attendue est de {{ min_height }}px.</target>
             </trans-unit>
             <trans-unit id="128">
                 <source>The video has too few pixels ({{ pixels }} pixels). Minimum amount expected is {{ min_pixels }} pixels.</source>
-                <target state="needs-review-translation">La vidéo a trop peu de pixels ({{ pixels }}). La quantité minimale attendue est {{ min_pixels }}.</target>
+                <target>La vidéo a trop peu de pixels ({{ pixels }}). La quantité minimale attendue est de {{ min_pixels }} pixels.</target>
```

Comment:
FTR, I removed the word "pixels" from all translation in order to not have to deal with plurals: the original sentence has two plurals, and our simple plural system cannot handle this. I made the translated sentences contain none.
Of course, I'm fine re-adding the word. Here, we make the assumption that min_pixel will always be higher than 1.